src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1254,11 +1213,14 @@ namespace WaterToAirHeatPumpSimple {
 								MixHumRat = OutAirFrac * FinalSysSizing( CurSysNum ).PrecoolHumRat + ( 1.0 - OutAirFrac ) * FinalSysSizing( CurSysNum ).RetHumRatAtCoolPeak;
 							}
 						}
+						// supply air condition is capped with that of mixed air to avoid SHR > 1.0
+						SupTemp = min( MixTemp, SupTemp );
+						SupHumRat = min( MixHumRat, SupHumRat );
 						OutTemp = FinalSysSizing( CurSysNum ).OutTempAtCoolPeak;
 						rhoair = PsyRhoAirFnPbTdbW( StdBaroPress, MixTemp, MixHumRat, RoutineName );
 						MixEnth = PsyHFnTdbW( MixTemp, MixHumRat );
 						MixWetBulb = PsyTwbFnTdbWPb( MixTemp, MixHumRat, StdBaroPress, RoutineName );
-						SupEnth = PsyHFnTdbW( SupTemp, SupHumRat );
+						SupEnth = PsyHFnTdbW( SupTemp, MixHumRat );
```

Comment:
My understanding is that in sensible cooling process the entering humidity ratio, which is MixHumRatio, remains constant.  For that reason, I chose to use entering condition. I believe that both should give similar results but not identical.